Kinvey/KinveyTests/DataTypeTestCase.swift

```diff
@@ -186,6 +186,60 @@ class DataTypeTestCase: StoreTestCase {
 
     }
     
+    func testDatePull() {
```

Comment:
I think instead of a test like `testDatePull()`, you should write a test that does basic operations on all our supported datatypes. This can go in a future PR.